apps/code-examples/src/app/code-examples/forms/radio/icon/demo.component.ts

```diff
@@ -8,11 +8,11 @@ import {
 } from '@angular/forms';
 import { SkyRadioModule } from '@skyux/forms';
 
-type Item = {
+interface Item {
```

Comment:
Types were changed to interfaces by a new eslint rule `@typescript-eslint/consistent-type-definitions`.